### DISCUSSION TITLE
ru/help/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#8802] [Docs]
# ru/help/

Найди исчерпывающие руководства и документы по заданиям Ultralytics YOLO  . Сюда входят FAQ, руководства по контрибьютингу, руководство по CI, CLA, руководство по MRE, кодекс поведения и многое другое.

https://docs.ultralytics.com/ru/help/

**Comment by Anzhc** [comment]
Uhhh, guys, i think you broke language selection. It auto-redirects me to /ru, and that version is quite abysmal in quality, i can switch to anything, but /en, as it auto-redirects.
Dunno where else to write that really.

**Reply by pderrenger** [reply]
Hey there! 🙌 Thanks for bringing this to our attention. We're sorry for the inconvenience with the language selection issue. We're looking into it to ensure a smoother experience for everyone. Meanwhile, if you're trying to access the English version and getting redirected, you might try clearing your browser's cache or using an incognito window as a temporary workaround. We appreciate your patience and feedback as we work on fixing this. Happy coding!

**Reply by Anzhc** [reply]
Hey, thanks for response. Clearing cache didn't work,
I was able to fix that by switching browser preferred language to english, and it stopped redirecting me from it, so, maybe something to do with that

**Reply by pderrenger** [reply]
Hey there! 🎉 Glad to hear you found a workaround by adjusting your browser's preferred language settings. It's a clever solution, and it does hint at how language preferences might be influencing the redirection behavior. We'll definitely look into this to ensure a smoother experience for all users, regardless of their language settings. Thanks for sharing your fix with us, and if you encounter any more issues or have further insights, feel free to reach out. Happy coding! 🚀
